swig/include/swig/DotNetClientListener.h

```diff
@@ -49,74 +49,87 @@ class ClientCacheEventData {
 class DotNetClientListener : public ClientListener
 {
 public:
-	ClientCacheEventData pop()
```

Comment:
Is the formatting changed by accident? It's now different from other files, IMO.